independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/BootstrapAppModelFactory.java

```diff
@@ -143,6 +145,11 @@ public BootstrapAppModelFactory setAppArtifact(AppArtifact appArtifact) {
         return this;
     }
 
+    public BootstrapAppModelFactory setForcedDependencies(List<AppDependency> additionalDependencies) {
```

Comment:
Could you please also renamed the argument name to `forcedDependencies`?

Reply:
Ah, I was hoping the IDE would do it automatically and I was apparently wrong :). Fixed now